keep-ui/utils/hooks/useProviders.ts

```diff
@@ -1,16 +1,16 @@
 import { useSession } from "next-auth/react";
 import { getApiURL } from "../apiUrl";
-import useSWR from "swr";
+import useSWR, { SWRConfiguration } from "swr";
 import { ProvidersResponse } from "app/providers/providers";
 import { fetcher } from "../fetcher";
 
-export const useProviders = () => {
+export const useProviders =  (options?: SWRConfiguration) => {
```

Comment:
Maybe it's good to add a default object here i.e. `{ revalidateOnFocus: false, revalidateOnMount: false }`?

Reply:
added